api4/user_test.go

```diff
@@ -690,6 +690,57 @@ func TestGetUsers(t *testing.T) {
 	CheckUnauthorizedStatus(t, resp)
 }
 
+func TestGetUsersWithoutTeam(t *testing.T) {
+	th := Setup().InitBasic().InitSystemAdmin()
+	defer TearDown()
+	Client := th.Client
+	SystemAdminClient := th.SystemAdminClient
+
+	if _, resp := Client.GetUsersWithoutTeam(0, 100, ""); resp.Error == nil {
+		t.Fatal("should prevent non-admin user from getting users without a team")
+	}
+
+	// These usernames need to appear in the first 100 users for this to work
+
+	user, resp := Client.CreateUser(&model.User{
+		Username: "a000000000" + model.NewId(),
+		Email:    "success+" + model.NewId() + "@simulator.amazonses.com",
+		Password: "Password1",
+	})
+	CheckNoError(t, resp)
+	LinkUserToTeam(user, th.BasicTeam)
+	defer app.Srv.Store.User().PermanentDelete(user.Id)
+
+	user2, resp := Client.CreateUser(&model.User{
+		Username: "a000000001" + model.NewId(),
+		Email:    "success+" + model.NewId() + "@simulator.amazonses.com",
+		Password: "Password1",
+	})
+	CheckNoError(t, resp)
+	defer app.Srv.Store.User().PermanentDelete(user2.Id)
+
+	if rusers, resp := SystemAdminClient.GetUsersWithoutTeam(0, 100, ""); resp.Error != nil {
+		t.Fatal(resp.Error)
+	} else {
```

Comment:
Can you use the pattern like this for the test?
```
<your-result>, resp := <your-driver-call>
CheckNoError(t, resp)

// do your result validation here
```

I'm trying to keep the pattern consistent through api4 tests and it also removes a level of nesting :)